hi-ml-histopathology/src/histopathology/configs/classification/BaseMIL.py

```diff
@@ -255,14 +255,16 @@ def get_dataloader_kwargs(self) -> dict:
             dataloader_kwargs = super().get_dataloader_kwargs()
         return dataloader_kwargs
 
-    def get_transform(self, image_key: str) -> Callable:
+    def get_transform(self, image_key: str) -> Dict[str, Callable]:
```

Comment:
The title of this method doesn't match what is now being returned

Reply:
not sure I understand this comment, is this about the signature or the method name? how would you like this changed?

Reply:
The method name - even just making it plural i.e. `get_transforms` makes it clearer that more than one transform is returned

Reply:
from the other comment I understand is about the name. I can rename all `transforms_dict` and `get_transforms_dict` - would that solve? I'd probably find it a bit redundant given we have full typing signature everywhere but happy to go for more clarity

Reply:
renamed now